tools/gen-intrinsics-specs.py

```diff
@@ -787,6 +792,8 @@ def get_intrinsics_db(path):
                         help="CSV file that map the intrinsics to their classification.", required=True)
     parser.add_argument("--outfile", metavar="<path>", type=str,
                         help="Output file where the RST of the specs is written.", required=True)
+    parser.add_argument("--format", metavar="<path>", type=str,
```

Comment:
Hum, instead of format, would you mind changing to a more specific name? Like `--table-syntax` or `--table-format`